src/main/java/mate/academy/Main.java

```diff
@@ -2,15 +2,16 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
+    private static final int THREAD_AMOUNT = 5;
+
     public static void main(String[] args) {
-        // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(THREAD_AMOUNT);
         AsyncRequestProcessor asyncRequestProcessor = new AsyncRequestProcessor(executor);
 
-        // Simulating multiple concurrent requests
-        String[] userIds = {"user1", "user2", "user3", "user1"}; // Note: "user1" is repeated
+        String[] userIds = {"user1", "user2", "user3", "user1"};
```

Comment:
Be cautious about using duplicate user IDs ('user1' appears twice). If the processing for a user ID should be unique, consider using a Set to eliminate duplicates before processing.